src/python/setup.py

```diff
@@ -173,7 +173,7 @@ def run(self):
         package_data={
             'turicreate': [
                 '_cython/*.so', '_cython/*.pyd',
-                '*.so', '*.dylib',
+                '*.so', '*.dylib', 'toolkits/*.so',
```

Comment:
Why is this change needed? Are we putting `.so` files in the `toolkits` folder now?